docs/sources/mimir/manage/mimir-runbooks/_index.md

```diff
@@ -1411,6 +1411,17 @@ How to **investigate** and **fix** it:
     - Check the number of in-memory series shown on the `Mimir / Tenants` dashboard for an approximation of the number of series that will be compacted once these blocks are shipped from ingesters.
     - Check the configured `compactor_split_and_merge_shards` for the tenant. A reasonable rule of thumb is 8-10 million series per compactor shard - if the number of series per shard is above this range, increase `compactor_split_and_merge_shards` for the affected tenant(s) accordingly.
 
+##### Persistent Volume Resizing Prequisite - store-gateways
+
+In case persistent volume resizing is needed for store-gateways, ensure the following steps are taken before proceeding, if automatic downscaling and/or autoscaling, have been enabled:
+
+- Disable automatic downscaling for store-gateways.
```

Comment:
Could you please be more specific about what should be done here?

Reply:
Hello @charleskorn . Thank you for reviewing. I believe the runbook is now more concise and specific. I also considered several comments before drafting it.

- Automatic downscaling is enabled only in the Jsonnet ([Ref](https://github.com/grafana/mimir/blob/main/operations/mimir/store-gateway-automated-downscale.libsonnet)). I couldn't find any reference to it in Helm, which is why I skipped the Helm reference. Please correct me if I’m wrong.
- Autoscaling in both [Jsonnet](https://github.com/grafana/mimir/blob/dc3ddfa730e9a1ddb2f74c34ade885ae7b6d245f/operations/mimir/multi-zone.libsonnet#L22) and [Helm](https://github.com/grafana/mimir/blob/dc3ddfa730e9a1ddb2f74c34ade885ae7b6d245f/operations/helm/charts/mimir-distributed/values.yaml#L2009) is similar. We configure a number based on a variable, which is then divided by 3 to determine the desired number of replicas for each StatefulSet in each zone. Therefore, I skipped it from the prerequisite list.

Let me know if it makes sense to you. Thank you